### ISSUE TITLE
[Feature Request] Animating transition in the detail view.

### ISSUE THREAD

**Issue by avikantz** [mamaral/xkcd-Open-Source#56]
Going forward or backward is currently bland. It'd be cool if we can *page* through the comics.

**Comment by mamaral** [comment]
Agreed - I have some ideas I'll be incorporating into the version after the one I'm submitting shortly. Thanks!
